### ISSUE TITLE
License for modification and redistribution

### ISSUE THREAD

**Issue by utsekaj42** [KTH-Nek5000/PipeMesh#1]
Hi, would it be possible to clarify if this code is available under some sort of open source license?

**Comment by pschlatt1** [comment]
Yes, assume that it is opensource. If you can use it, you can also reference the corresponding work where this code was used. This is DOI: https://doi.org/10.1017/jfm.2017.749

**Comment by utsekaj42** [comment]
Great, I'd still suggest/request a formal license file/public domain statement be added as it would clarify this and prevent any unlikely legal issues/concerns. https://choosealicense.com/no-permission/
